tests/compose/test-misc-tweaks.sh

```diff
@@ -10,11 +10,13 @@ treefile_append "repos" '["test-repo"]'
 # test `recommends: true` (test-basic[-unified] test the false path)
 build_rpm foobar recommends foobar-rec
 build_rpm foobar-rec
+build_rpm quuz
+build_rpm corge
 
 echo gpgcheck=0 >> yumrepo.repo
 ln "$PWD/yumrepo.repo" config/yumrepo.repo
 # the top-level manifest doesn't have any packages, so just set it
-treefile_append "packages" '["foobar"]'
+treefile_append "packages" $'["\'foobar >= 0.5\' quuz \'corge < 1.0.0\'"]'
```

Comment:
The quoting required here is... :exploding_head: 
It has to get through bash, then Python, then YAML, then rpm-ostree...